### DISCUSSION TITLE
Upgrade from v2.23.1 to v2.26.2 and it broke next.js webpack config!?

### DISCUSSION THREAD

**Discussion by meandillar** [TanStack/query#1300] [General] [answered]
Hi all,
I'm using react-query with next.js and I have a custom webpack config in next.config.js which updates my css class names like so:

```
webpack(config) {
    config.module.rules[1].oneOf.forEach((moduleLoader, i) => {
      if (moduleLoader.test && moduleLoader.test.toString() === moduleLoaderTest.toString()) {
        Array.isArray(moduleLoader.use) &&
          moduleLoader.use.forEach(l => {
            if (l.loader.includes('/css-loader')) {
              l.options = {
                ...l.options,
                modules: {
                  ...l.options.modules,
                  getLocalIdent: getCSSModuleLocalIdent, // function to alter class names
                },
              }
            }
          })
      }
    })
    config.node = { fs: 'empty' }
    return config
  }
```

I upgraded react-query to the latest version and the module rules changed shape. The css loaders are found on config.module.rules[2] instead of config.module.rules[1] and css-loader has disappeared. Downgraded back to v2.23.1 again and it fixes the problem.

Seems odd that react-query should be affecting my webpack config. Does anyone have an idea of why this would happen?

**Comment by tannerlinsley** [comment, marked as answer]
There's literally nothing in React Query that could affect this, so I would suggest nuking your node_modules and trying again.
